crates/parry3d/benches/all.rs

```diff
@@ -1,6 +1,8 @@
 #![feature(test)]
```

Comment:
Due to this feature, we can only use nightly. I'm not sure if this is avoidable.

Reply:
Yeah, not sure either. Running benchmarks in nightly is fine.